pyseir/cli.py

```diff
@@ -145,7 +143,8 @@ def _build_all_for_states(
             output_interval_days=output_interval_days,
             output_dir=output_dir,
         )
-        p.map(states_only_func, states)
+        states_regions = map(pipeline.Region.from_state, states)
```

Comment:
small, but I generally prefer a list comprehension to map #pythonic

Reply:
Done. The simplicity of map is too tempting when I have a function and a list and don't need to do anything but put them together.